runtime/System/Console/Hawk/Representable.hs

```diff
@@ -24,14 +24,14 @@ module System.Console.Hawk.Representable (
 
 ) where
 
-import Prelude
-import Data.ByteString.Lazy.Char8 (ByteString)
+import           Data.ByteString.Lazy.Char8 (ByteString)
 import qualified Data.ByteString.Lazy.Char8 as C8 hiding (hPutStrLn)
-import qualified Data.List as L
-import Data.Set (Set)
-import qualified Data.Set as S
-import Data.Map (Map)
-import qualified Data.Map as M
```

Comment:
I like the `qualified` alignment, but the `as` alignment looks really weird. I assume it's done automatically by your editor?